modules/WebPublishPlugin/src/main/java/net/clementlevallois/web/publish/plugin/controller/GephiPluginDesktopLogic.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * author: Clément Levallois
+ */
+package net.clementlevallois.web.publish.plugin.controller;
+
+/*
+ *
+ * @author LEVALLOIS
+ */
+import java.util.ResourceBundle;
+import org.gephi.desktop.io.export.spi.ExporterClassUI;
+import org.openide.DialogDisplayer;
+import org.openide.NotifyDescriptor;
+import org.openide.util.NbBundle;
+import org.openide.util.lookup.ServiceProvider;
+
+/*
+
+ * @author Clement Levallois
+ */
+@ServiceProvider(service = ExporterClassUI.class)
```

Comment:
You may want to add a position to make sure it always appear third in the menu, after the default options. position=200 or something like this.

Reply:
✅